src/migrations/1740372108538-add-gallery-type.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddGalleryType1740372108538 implements MigrationInterface {
+    name = 'AddGalleryType1740372108538'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TYPE "public"."gallery_type_enum" AS ENUM('Real', 'Virtual')`);
+        await queryRunner.query(`ALTER TABLE "gallery" ADD "type" "public"."gallery_type_enum" NOT NULL`);
```

Comment:
The type is not null, but what value will be set by default to the existing galleries?